Specify deprecated p2p types for `shmem_wait_until` and `shmem_test` (#14)
naveen-rn/specification: content/shmem_test.tex

int @\FuncDecl{shmem\_test}@(TYPE *ivar, int cmp, TYPE cmp_value);
\end{C11synopsis}
where \TYPE{} is one of the standard \ac{AMO} types specified by

> Table \ref{stdamotypes}
> \begin{DeprecateBlock}
> and where \TYPE{} is one of \{\CTYPE{short}, \CTYPE{unsigned short}\}
> point-to-point synchronization types
> specified by Table~\ref{p2psynctypes}.

Reviewer: ```suggestion
Table \ref{stdamotypes},
\begin{DeprecateBlock}
or \TYPE{} is one of \{\CTYPE{short}, \CTYPE{unsigned short}\} and
has a corresponding \TYPENAME{} specified by Table~\ref{p2psynctypes}.
```

\end{DeprecateBlock}

\begin{Csynopsis}
int @\FuncDecl{shmem\_\FuncParam{TYPENAME}\_test}@(TYPE *ivar, int cmp, TYPE cmp_value);
\end{Csynopsis}
where \TYPE{} is one of the standard \ac{AMO} types and has a

> corresponding \TYPENAME{} specified by Table~\ref{stdamotypes}
> \begin{DeprecateBlock}
> and where \TYPE{} is one of \{\CTYPE{short}, \CTYPE{unsigned short}\}
> point-to-point synchronization types and has a
> corresponding \TYPENAME{} specified by Table~\ref{p2psynctypes}.

Reviewer: ```suggestion
corresponding \TYPENAME{} specified by Table~\ref{stdamotypes},
\begin{DeprecateBlock}
or \TYPE{} is one of \{\CTYPE{short}, \CTYPE{unsigned short}\} and
has a corresponding \TYPENAME{} specified by Table~\ref{p2psynctypes}.
```
